frontend/stylesheets/_dashboard.less

```diff
@@ -370,3 +370,14 @@
     }
   }
 }
+
+pre.dataset-import-folder-structure-hint {
+  text-align: left;
+  display: inline-block;
+  line-height: 16px;
+  color: #616161;
```

Comment:
In preparation of the dark theme: Could you express these colors with antd-provided variables?
- https://antdtheme.com/default#Color
- https://ant.design/docs/spec/colors